tests/affiliatedcertification/tests/affillated_certification_helm_chart.go

```diff
@@ -0,0 +1,107 @@
+package tests
+
+import (
+	"os/exec"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	tsparams "github.com/test-network-function/cnfcert-tests-verification/tests/affiliatedcertification/parameters"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalhelper"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalparameters"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/execute"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/namespaces"
+)
+
+var _ = Describe("Affiliated-certification helm chart certification,", func() {
+	execute.BeforeAll(func() {
+		By("Define tnf config file")
+		err := globalhelper.DefineTnfConfig(
+			[]string{tsparams.TestHelmChartCertified},
+			[]string{tsparams.TestPodLabel},
+			[]string{},
+			[]string{})
+		Expect(err).ToNot(HaveOccurred(), "error defining tnf config file")
+		err = namespaces.Create(tsparams.TestHelmChartCertified, globalhelper.GetAPIClient())
+		Expect(err).ToNot(HaveOccurred(), "Error creating namespace")
+	})
```

Comment:
Because the namespace is now being fully deleted after each test case it needs to be re-created before each case
```suggestion
	})
	
	BeforeEach(func() {
		By("Create namespace")
		err := namespaces.Create(tsparams.TestHelmChartCertified, globalhelper.GetAPIClient())
		Expect(err).ToNot(HaveOccurred(), "Error creating namespace")
	})
```